vessim/core.py

```diff
@@ -108,11 +98,57 @@ def __init__(
 
         self._fill_method = fill_method
 
+    @classmethod
+    def from_dataset(
+        cls,
+        dataset: Union[str, Dict],
+        data_dir: Union[str, Path] = ".",
+        scale: float = 1.0,
+        start_time: Optional[DatetimeLike] = None,
+        use_forecast: bool = True,
+    ):
+        """Retrieves a TimeSeriesApi from a dataset.
+
+        Args:
+            dataset: If a string is provided, the TimeSeriesApi is loaded from one of the
+                vessim datasets. Currently available datasets are:
+                    `solcast2022_germany` and `solcast2022_global`
+                Otherwise, it should be a Dictionary containing info about the dataset
+                with following entries:
+                    `actual`: Name of the file containing the actual data.
+                    `forecast`: Name of the file containing the forecasted data. This is
+                        not needed if use_forecast is set to False.
+                    `fill_method`: The fill_method of the TimeSeriesApi. If not specified,
+                        `bfill` is used.
+                    `static_forecast`: Bool indicating if the forecast is static. If set
+                        to True, the forecast does not contain a `Request Timestamp`, but
+                        if not specified, the forecast is treated as non-static forecast.
+                        This is not needed if use_forecast is set to False.
+                    `url`: String with a URL to a zip-file if data not locally available.
```

Comment:
why should I go for `TimeSeriesApi.from_dataset(dataset={*kwargs})` instead of `TimeSeriesApi(*kwargs)`?